configuration/integrations/mailerlite.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "MailerLite"
+---
+
+The Rownd integration for MailerLite makes it easy to keep your subscriber list in sync. The integration will automatically create a new subscriber or update an existing one in your MailerLite account.
+
+#### Setup
+
+[The MailerLite integration can be enabled from the Rownd dashboard.](https://app.rownd.io)
+
+1. From the menu, select **Integrations.**
+
+2. Select the **Connector catalog** tab.
+
+3. From the list of available connectors, click on the **MailerLite**
+   entry. The connector setup dialog will appear.
+
+4. Enter a descriptive name and click **Next**.
+
+  ![Create the mailerlite integration](/images/integration_mailerlite_setup.png)
+
+#### Authenticate
+
+Enter your MailerLite API key. You can create or retrieve your API key from your MailerLite account.
+
+1. In your MailerLite account, navigate to the Integrations page.
```

Comment:
Can we link `MailerLite account` or `Integrations` to the right MailerLite URL where they can find these settings?

Reply:
I updated "Integrations" with the correct MailerLite link